src/material/datepicker/calendar-body.ts

```diff
@@ -159,18 +165,31 @@ export class MatCalendarBody implements OnChanges, OnDestroy, AfterViewChecked {
   /** Width of an individual cell. */
   _cellWidth: string;
 
+  private _didDragSinceMouseDown = false;
+
```

Comment:
Random thought: currently there's no special indication that a cell is being dragged. Maybe the Material-Design-ey thing to do would be to have an elevation box shadow on it?

Reply:
FWIW the internal spec that inspired this behavior does not add an elevation. It would be possible to add something like that later down the line if the material spec had something like it, but I'm reluctant to make any guesses about it now.

Reply:
Alright, let's leave it for now.